feedback.html

```diff
@@ -21,14 +21,14 @@
         <a href="#" class="nav__logo" data-aos="fade-right">Tourguide<span>.</span></a>
 
         <ul class="navLinks">
-          <li class="link home" data-aos="fade-down"><a href="#Home">Home</a></li>
-          <li class="link" data-aos="fade-down"><a href="#AboutUs">AboutUs</a></li>
-          <li class="link" data-aos="fade-down"><a href="#Destinations">Destinations</a></li>
-          <li class="link" data-aos="fade-down"><a href="#trip">Pricing</a></li>
-          <li class="link" data-aos="fade-down"><a href="#ReviewGallery">Trip Gallery</a></li>
-          <li class="link" data-aos="fade-down"><a href="#testimonials">Testimonials</a></li>
-          <li class="link" data-aos="fade-down"><a href="feedback.html">Rate Us?</a></li>
-          <li class="link" data-aos="fade-down"><a href="#contact">Contact Us</a></li>
+          <li class="link home" data-aos="fade-down"><a href="./index.html#home" target="_blank">Home</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#AboutUs" target="_blank">AboutUs</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#Destinations" target="_blank">Destinations</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#trip" target="_blank">Pricing</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#ReviewGallery" target="_blank">Trip Gallery</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#testimonials" target="_blank">Testimonials</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.htmlfeedback.html" target="_blank">Rate Us?</a></li>
+          <li class="link" data-aos="fade-down"><a href="./index.html#contact" target="_blank">Contact Us</a></li>
 
         </ul>
```

Comment:
@SadafKausar2025  

![image](https://github.com/apu52/Travel_Website/assets/114172928/c4bb750d-2cdc-4a7b-ad2e-ecc5dac7e698)

There is an unexpected error in the login page . Whenever you click on the Login button in the navbar of  the Feedback page this bug is showing . Please try to solve this issues

Reply:
Ok I will correct it